File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/ui/feeds/favorite/FavoriteStoriesViewModel.java
package com.mowdowndevelopments.blurb.ui.feeds.favorite;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.LiveData;

import com.mowdowndevelopments.blurb.database.BlurbDb;
import com.mowdowndevelopments.blurb.database.entities.Story;
import com.mowdowndevelopments.blurb.ui.feeds.BaseFeedViewModel;

import java.util.List;


public class FavoriteStoriesViewModel extends BaseFeedViewModel {
    final LiveData<List<Story>> storyList;

    public FavoriteStoriesViewModel(@NonNull Application app) {
        super(app);
        storyList = BlurbDb.getInstance(app).blurbDao().getStarredStoryList();
    }
}

File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/ui/feeds/favorite/FavoriteStoriesViewModel.kt
package com.mowdowndevelopments.blurb.ui.feeds.favorite

import android.app.Application
import androidx.lifecycle.LiveData
import com.mowdowndevelopments.blurb.database.BlurbDb.Companion.getInstance
import com.mowdowndevelopments.blurb.database.entities.Story
import com.mowdowndevelopments.blurb.ui.feeds.BaseFeedViewModel

class FavoriteStoriesViewModel(app: Application) : BaseFeedViewModel(app) {
    @JvmField
    val storyList: LiveData<List<Story>> = getInstance(app).blurbDao().getStarredStoryList()

}